Automattic/woocommerce-payments
# Fix modal redirection after classname change

#### Changes proposed in this Pull Request

#6250 was tested and approved before #6061 git merged. This second PR changed a class name which broke the first one, making the redirection stop working:

![image](https://github.com/Automattic/woocommerce-payments/assets/12385501/9c78a3d4-8faa-407c-8fb8-067b3ab09f49)

This PR fixes that by updating the expected classname.


#### Testing instructions

Make sure you don't get stuck in this page when trying to use WooPay and you get redirected.

![image](https://github.com/Automattic/woocommerce-payments/assets/12385501/81dce785-1fb0-408f-82e9-a446120a360c)


-------------------


**Post merge**
